templates/statsd_mapping.conf.erb

```diff
@@ -0,0 +1,8 @@
+<% @real_statsd_maps.sort_by!{ |metric| metric[:name] }.each do |metric| -%>
+<%= metric['map'] %>
+name="<%= metric['name'] %>"
+<% metric['labels'].sort.each do |key, value| -%>
+<%= key %>="<%= value %>"
+<% end -%>
+
```

Comment:
I think that this blank line is redundant given that the previous line ends with `-%>`